addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/SolarEdgeBindingConstants.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SolarEdgeBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Alexander Friese - Initial contribution
+ */
+public class SolarEdgeBindingConstants {
+
+    private static final @NonNull String BINDING_ID = "solaredge";
```

Comment:
Please do not use NonNull but use NonNullByDefault as [described in the coding guidelines](https://www.openhab.org/docs/developer/development/guidelines.html#a-code-style)

Reply:
fixed it